src/Content/AttachmentLocal.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Phlib\Mail\Content;
+
+use Phlib\Mail\Exception\RuntimeException;
+
+/**
+ * Attachment class used to add local files to a Mail object
+ *
+ * @package Phlib\Mail
+ */
+class AttachmentLocal extends Attachment
```

Comment:
Yeah, so `Attachment::createFromFile($filename)` instead makes sense. Only downside is that the file content would have to be loaded on construct, whereas at the moment it's left until the call to `getContent()`. Do you think that's OK? If so I'll make the change.